extensions/amp-next-page/1.0/service.js

```diff
@@ -840,52 +866,160 @@ export class NextPageService {
   /**
    * Renders the template inside the separator element using
    * data from the current article (if a template is present)
+   * otherwise rehydrates the default separator
    *
    * @param {!Element} separator
    * @param {!Page} page
    * @return {!Promise}
    */
   maybeRenderSeparatorTemplate_(separator, page) {
-    if (!this.templates_.hasTemplate(separator)) {
+    if (!this.hasDefaultSeparator_ && !this.templates_.hasTemplate(separator)) {
       return Promise.resolve();
     }
+
+    // Re-render templated separator (if needed)
+    return this.getSeparatorContent_(separator, page).then(rendered => {
+      return this.mutator_.mutateElement(separator, () => {
+        removeChildren(dev().assertElement(separator));
+        separator.appendChild(rendered);
+      });
+    });
+  }
+
+  /**
+   * Creates the internal separator content based on the
+   * given information about the next page
+   * @param {!Element} separator
+   * @param {!Page} page
+   * @return {!Promise}
+   */
+  getSeparatorContent_(separator, page) {
     const data = /** @type {!JsonObject} */ ({
       title: page.title,
       url: page.url,
       image: page.image,
     });
-    return this.templates_
-      .findAndRenderTemplate(separator, data)
-      .then(rendered => {
-        return this.mutator_.mutateElement(separator, () => {
-          removeChildren(dev().assertElement(separator));
-          separator.appendChild(rendered);
-        });
-      });
+
+    // In the case of a templated separator, render the internal template
+    if (this.templates_.hasTemplate(separator)) {
+      return this.templates_.findAndRenderTemplate(separator, data);
+    }
+
+    // Otherwise this is a default separator
+    devAssert(this.hasDefaultSeparator_);
+
+    const content = this.doc_.createElement('div');
+    content.classList.add('amp-next-page-separator-content');
+
+    const image = this.doc_.createElement('img');
+    image.classList.add('amp-next-page-separator-img');
+    image.src = data['image'];
+    content.appendChild(image);
+
+    const title = this.doc_.createElement('span');
+    title.classList.add('amp-next-page-separator-title');
+    title.textContent = `Next article: ${data['title']}`;
+    content.appendChild(title);
+
+    return Promise.resolve(content);
```

Comment:
This all goes away since we're using a simple line separator now. However, for the `Promise`, it's for testing. If there's a better way to do it please let me know! (Essentially not sure how to wait for a promised spy/stub).